arctic/_util.py

```diff
@@ -15,6 +17,24 @@
 _use_new_count_api = None
 
 
+class FwPointersCfg(Enum):
+    ENABLED = 0
+    DISABLED = 1
+    HYBRID = 2
+
+
+# Forward pointers configuration
+FW_POINTERS_KEY = 'CHUNK_IDS'
+ARCTIC_FORWARD_POINTERS_RECONCILE = bool(os.environ.get('ARCTIC_FORWARD_POINTERS_RECONCILE'))
```

Comment:
Worth documenting these environment variables. Not sure what `...RECONCILE` does without reading the code

Reply:
yep, adding comments to make it clear.

In the next PR for the async, will move all env variable configs to a separte arctic/config.py module, because they have now become too many and are spread in many modules.

Reply:
@yschimke  @jamesblackburn  @bmoscon  I have now added comments with explanation for variables and the fw pointer helper methods.

I will mark this specific conversation now as resolved.